plugins/usd/module/vtkF3DUSDImporter.cxx

```diff
@@ -539,8 +541,20 @@ class vtkF3DUSDImporter::vtkInternals
     pxr::UsdSkelBakeSkinning(this->Stage->Traverse());
     this->Stage->Save();
 
-    vtkNew<vtkMatrix4x4> identity;
-    this->ImportNode(renderer, this->Stage->GetPseudoRoot(), pxr::SdfPath("/"), identity);
+    vtkNew<vtkMatrix4x4> rootTransform;
+
+    pxr::TfToken up = pxr::UsdGeomGetStageUpAxis(this->Stage);
+
+    if (up == pxr::UsdGeomTokens->z)
```

Comment:
so it can only be +Z or +Y ? No other possibilities ?

Or maybe I dont get what this condition achieve.

Reply:
Yes, no other possibilities. Here's the documentation about it:

> The stage up axis is encoded as stage metadatum upAxis, whose legal values are "Y" and "Z", as represented by UsdGeomTokens->y and UsdGeomTokens->z.

Reply:
alright. You mentionned wanting to add some king of logging, lets add it ?